### ISSUE TITLE
Example of use without node.js

### ISSUE THREAD

**Issue by yurikaz** [mfogel/polygon-clipping#47]
Thanx a lot for you lib
Please can you give a small example of using it without node.js
We have "snake" track and we need to calculate interselections and show it.

**Comment by yurikaz** [comment]
Thanx for your answer.
When I am doing something lile this I have an error
"Uncaught ReferenceError: polygon is not defined"

**Comment by yurikaz** [comment]
I think if we change in js "polygon-clipping" to "polygonclipping" it will work

**Comment by rowanwins** [comment]
ah yes of course the hyphen isn't ideal, that said as a temporary work around you can do something like `window['polygon-clipping'].union(poly1, poly2)`

**Comment by mfogel** [comment]
Agreed, it would be good to add to the README quickstart instructions for use directly in the browser. I'll try to fit this into the next non-bug-fix-focused release.
